src/streams.c

```diff
@@ -1582,17 +1582,31 @@ Obj FuncPOSITION_FILE (
     Obj             fid )
 {
     Int             ret;
-
     /* check the argument                                                  */
     while ( ! IS_INTOBJ(fid) ) {
         fid = ErrorReturnObj(
             "<fid> must be an integer (not a %s)",
             (Int)TNAM_OBJ(fid), 0L,
             "you can replace <fid> via 'return <fid>;'" );
     }
-    
-    ret = SyFtell( INT_INTOBJ(fid) );
-    return ret == -1 ? Fail : INTOBJ_INT(ret);
+
+    Int ifid = INT_INTOBJ(fid);
+
+
```

Comment:
In case you end up making other changes: perhaps remove second empty line? or even both empty lines, and then for symmetry change `ret = ...` to `Int ret = ...`, and remove the declaration of `ret` above?